pulser-core/pulser/channels/channels.py

```diff
@@ -11,20 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""The various hardware channel types."""
+"""Defines the Channel ABC and its subclasses."""
```

Comment:
This file is very long. Wouldn't be better to try splitting it at least into two parts?

Reply:
Sure, I guess I can move the subclasses to another file.